### DISCUSSION TITLE
ALWAYS show labels on ALL buttons on the task bar

### DISCUSSION THREAD

**Discussion by pbakes** [Open-Shell/Open-Shell-Menu#597] [Ideas]
Like when your "Combine taskbar buttons" setting is set to Never, except that EVERY program icon has a label.

Is that possible?

**Comment by bonzibudd** [comment]
Do you mean static buttons, like items pinned to the taskbar?

**Reply by pbakes** [reply]
Yes, all pinned items. I just checked DisplayFusion and Actual Window Manager and neither of them have that capability; the ability to display both your pinned items and the system tray on all taskbars.
